hyperspy/_signals/signal2d.py

```diff
@@ -22,7 +22,11 @@
 import dask.array as da
 import scipy as sp
 import logging
-from skimage.feature.register_translation import _upsampled_dft
+try:
+    # For scikit-image >= 0.17.0
+    from skimage.registration._phase_cross_correlation import _upsampled_dft
+except ModuleNotFoundError:
```

Comment:
Why not `ImportError`?

Reply:
This module doesn't exist in previous version.

Reply:
More that `ModuleNotFoundError` is a sub-class of `ImportError` but not that it really matters.

Reply:
Looks good to go!